tests/unit/base/test_deprecation.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+import warnings
+
+from twilio.base.obsolete import deprecated_method
+
+
+class DeprecatedMethodTest(unittest.TestCase):
+
+    def test_deprecation_decorator(self):
+
+        @deprecated_method()
+        def old_method():
+            return True
+
+        with warnings.catch_warnings(record=True) as caught_warnings:
+            warnings.simplefilter("always")
+            # Call function that should raise a warning, but still execute
+            self.assertTrue(old_method())
+            if len(caught_warnings):
```

Comment:
Should replace this check with an assertion.

Reply:
Done